ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/Simple_Renderer/Simple_Renderer.cpp

```diff
@@ -76,7 +76,7 @@ void Simple_Renderer::componentComplete()
     m_mapView->setMap(m_map);
 
     // set viewpoint using the two farthest points as an envelope with padding
```

Comment:
(this would be optional for readability) could change comment to "with 50m padding" or maybe create a local var called "paddingM = 50;"?